### PR TITLE
Issue #11214: Specified Violation messages in AbstractJavadocCheck

### PR DESCRIPTION
**Pull request by sd9616**
Issue #11214

### CODE REVIEW

**Comment by sd9616** [comment]
I'm not sure if I can/should fix these CI failures as they seem to come from changes made by this [commit](https://github.com/checkstyle/checkstyle/commit/2cb09d6fc2858fce332a3d0c0eed3066e2da423c)

**Comment by romani** [comment]
Please review CI failures

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/abstractjavadoc/InputAbstractJavadocNonTightHtmlTags.java [hunk 1]
```diff
@@ -8,35 +8,39 @@
...
  * @see "https://www.w3.org/TR/html51/syntax.html#optional-start-and-end-tags"
  */
 public class InputAbstractJavadocNonTightHtmlTags {
-    /** <p> <p> paraception </p> </p> */ // violation
+    /** <p> <p> paraception </p> </p> */ // violation 'Unclosed HTML tag found: p'
     private int field1;
 
     /**<li> paraTags should be opened</p> list isn't nested in parse tree </li>*/ // violation
+    // 'Unclosed HTML tag found: li'
     private int field2;
```
Comment (romani) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/abstractjavadoc/InputAbstractJavadocNonTightHtmlTags.java:23]: Please use `// violation above 'message'` syntax.
We need violation trailing comment to be with message on same line.

Apply this to your changes.
Please keep violation comment outside of javadoc.

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/abstractjavadoc/InputAbstractJavadocNonTightHtmlTags.java (hunk 1 above)
Comment (sd9616) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/abstractjavadoc/InputAbstractJavadocNonTightHtmlTags.java:23]: Done

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/abstractjavadoc/InputAbstractJavadocJavadocTagsWithoutArgs.java [hunk 2]
```diff
@@ -59,14 +63,16 @@ private void readObject(java.io.ObjectInputStream inputStream) {
...
+     * {@link} // violation 'Javadoc comment at column 13 has parse error. Details: mismatched
+     * input '}' expecting {LEADING_ASTERISK, WS, NEWLINE}" while parsing JAVADOC_INLINE_TAG'
      */
     public void method7() {
 
     }
 
     /**
-     * {@linkplain } // violation
+     * {@linkplain } // violation 'Javadoc comment at column 19 has parse error. Details: no viable
```
Comment (romani) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/abstractjavadoc/InputAbstractJavadocJavadocTagsWithoutArgs.java:74]: All `// violation` shout be outside of multiline comment `/**`

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/abstractjavadoc/InputAbstractJavadocNonTightHtmlTags.java (hunk 1 above)
Comment (romani) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/abstractjavadoc/InputAbstractJavadocNonTightHtmlTags.java:23]: Not done, please do for all your changes

**Comment by romani** [comment]
Reply by comment to let you double check yourself that it is done.

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/abstractjavadoc/InputAbstractJavadocNonTightHtmlTags.java (hunk 1 above)
Comment (sd9616) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/abstractjavadoc/InputAbstractJavadocNonTightHtmlTags.java:23]: Sorry about that, I was having some issues with committing my changes. 
I've double-checked now and  I believe it is done.

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/abstractjavadoc/InputAbstractJavadocJavadocTagsWithoutArgs.java (hunk 2 above)
Comment (romani) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/abstractjavadoc/InputAbstractJavadocJavadocTagsWithoutArgs.java]: @sd9616 , did you finish?

Comment (sd9616) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/abstractjavadoc/InputAbstractJavadocJavadocTagsWithoutArgs.java]: Yes, it is done

**Review by romani — APPROVED** [review]
Ok to merge.
Please help to fix same test at https://github.com/checkstyle/checkstyle/issues/11446

So it will be clear that you placed violation comment slightly not at right place.
But we still do good step forward.

**Pull request merged**